src/features/resume-review.ts

```diff
@@ -25,34 +36,55 @@ const sendResumeMessage = async (msg: Message): Promise<true | void> => {
   );
   if (!builder) {
     logger.log("[RESUME]", "Failed to generate resume PDF in thread");
-    return;
+    return [];
   }
 
-  await msg.channel.send({
-    files: builder,
-  });
-
-  return true;
+  return builder;
 };
 
 const resumeReviewPdf: ChannelHandlers = {
-  handleMessage: async ({ msg }) => {
-    // NOTE: This cast is safe as we are fetching the actual message in the index.ts/handleMessage
-    const message = msg as Message;
-    const cooldownKey = `resume-${msg.channelId}`;
+  handleMessage: async ({ msg: message }) => {
+    if (
+      !message.channel.isThread() ||
+      message.channel.parentId !== CHANNELS.resumeReview
+    ) {
+      return;
+    }
+    const cooldownKey = `resume-${message.channelId}`;
 
     if (cooldown.hasCooldown(message.author.id, cooldownKey)) {
+      message.channel.send(
+        "You posted < 5 minutes ago, not creating a new preview",
```

Comment:
```suggestion
        "You posted just a few minutes ago. Please wait a bit before creating a new preview.",
```

Let's make it a bit friendlier 😄

Reply:
:grumpy: